whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/MntLowerAttributeValidationSpec.groovy

```diff
@@ -457,12 +459,14 @@ class MntLowerAttributeValidationSpec extends BaseQueryUpdateSpec {
         )
 
         then:
-        ack.success
+        ack.failed
```

Comment:
modify inetnum (assigned pi and lir maintained): **add mnt-lower** of lir mnt by lir